packages/mcp-provider-devops/src/tools/checkCommitStatus.ts

```diff
@@ -0,0 +1,94 @@
+import { z } from "zod";
+import { CallToolResult } from "@modelcontextprotocol/sdk/types.js";
+import { McpTool, McpToolConfig, ReleaseState, Toolset, TelemetryService } from "@salesforce/mcp-provider-api";
+import axios from "axios";
+import { getConnection } from "../shared/auth.js";
+
+const inputSchema = z.object({
+  username: z.string().describe("Username of the DevOps Center org"),
+  requestId: z.string().describe("Request ID from the commit operation to check status for")
+});
+type InputArgs = z.infer<typeof inputSchema>;
+type InputArgsShape = typeof inputSchema.shape;
+type OutputArgsShape = z.ZodRawShape;
+
+export class CheckCommitStatus extends McpTool<InputArgsShape, OutputArgsShape> {
+  private readonly telemetryService: TelemetryService;
+
+  constructor(telemetryService: TelemetryService) {
+    super();
+    this.telemetryService = telemetryService;
+  }
+
+  public getReleaseState(): ReleaseState {
+    return ReleaseState.GA;
+  }
+
+  public getToolsets(): Toolset[] {
+    return [Toolset.OTHER];
+  }
+
+  public getName(): string {
+    return "check_commit_status";
```

Comment:
in yesterday's meeting we said these names are too generic and might overlap with IDE's built-in/other MCP tools.

Did John/CX had any feedback on this?

the other conflicting ones are:
*  `create_pull_request`
* `list_projects`
* `detect_conflict`
* `resolve_conflict`